test/invoker.ts

```diff
@@ -34,7 +34,8 @@ describe('request to HTTP function', () => {
       (req: express.Request, res: express.Response) => {
         res.send(req.body.text.toUpperCase());
       },
-      invoker.SignatureType.HTTP
+      invoker.SignatureType.HTTP,
+      '/'
```

Comment:
This is an odd function target. I suggest something like 'testFunction'.

Reply:
Applied suggestion of `testFunction`.